x-pack/platform/plugins/shared/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.tsx

```diff
@@ -417,6 +419,12 @@ export const RulesList = ({
 
   const handleClearRuleParamFilter = () => updateFilters({ filter: 'ruleParams', value: {} });
 
+  useEffect(() => {
+    if (onPageReady && !rulesState?.isLoading) {
+      onPageReady();
+    }
+  }, [onPageReady, rulesState?.isLoading]);
+
```

Comment:
Wouldn't be better to use [usePageReady](https://github.com/cesco-f/kibana/blob/main/src/platform/packages/shared/kbn-ebt-tools/src/performance_metrics/context/use_page_ready.ts)?

It also already handles calling `onPageRefreshStart` when needed.

Reply:
Probably yes, thanks I've forgotten about this one.